server/lib/user/user.getByTelegramUserId.js

```diff
@@ -14,13 +14,25 @@ const { NotFoundError } = require('../../utils/coreErrors');
 async function getByTelegramUserId(telegramUserId) {
   const user = await db.User.findOne({
     where: { telegram_user_id: telegramUserId },
-    attributes: ['id', 'language'],
-    raw: true,
```

Comment:
No raw anymore?

Reply:
I just .get({ plain: true }); after to get the created_at as native JS object like in the getUserById